src/index.html

```diff
@@ -7,12 +7,60 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <section
+      data-qa="card"
+      class="card"
+    >
+      <div class="card__image"></div>
```

Comment:
Yes, you are correct! If an image provides important information about a product, it should be included using the `<img>` tag in HTML. This allows the image to be properly displayed on a webpage. Additionally, you can use the alt attribute to provide a textual description of the image for accessibility purposes and for situations where the image might not load.